README.md

```diff
@@ -65,7 +73,7 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
 
-- All the files necessary for deployment are in the /dist folder
+- All the files necessary for deployment are in the `/dist` folder
```

Comment:
- [ ] In `src/status.js` kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.

Reply:
I really do not understand your first comment about the README file. I do have
- All the files necessary for deployment are in the `/dist` folder stated there! Please, clarify!
For your second comment I would say:
- I used arrow functions where I thought is appropriate and useful. For example in `checkButton` function. There is no specific requirements to have all my regular functions to be arrow functions! Please, have a look at project requirements from my [README.md](https://github.com/Hombre2014/ToDoList/tree/interactive#readme) file (step 2).
- In [Java Best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) article from Microverse there is no rule to have all regular functions as arrow functions.
- In [code clean](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) article cited above also there is nothing stating that you should use only arrow functions!
- I am repeating the week and this same branch (code) have been approved last week. You can see it [here](https://github.com/Hombre2014/ToDoList/pull/2#pullrequestreview-821188860).
So, I do not really understand your rejection. These might be recommendations and optional changes, but rejection? Please, reply.